myo_multithreading_examp.py

```diff
@@ -41,4 +42,4 @@ def print_battery(bat):
 
 	except KeyboardInterrupt:
 		print("Quitting")
-		quit()
\ No newline at end of file
+		quit()
```

Comment:
Param needs this
Use arrow functions to close over local variables. It is handy when you need to pass additional data to an event handler. 

See Supporting Document for considerations, though.

📄 [Arrow functions in methods](https://app.semasoftware.com/guides/60f79106b6aa6d1ebb73b2ac/arrow-functions-in-methods)

---
**Sema Reaction:** :trophy: This code is awesome | **Sema Tags:** Readable, Secure, Efficient

Reply:
Why did you do this? unreadable, unmaintainable, secure

---
**Sema Reaction:** :question: I have a question | **Sema Tags:** Unreadable, Secure, Maintainable

Reply:
hey this one is really awesome elegant 

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.svg)](https://semasoftware.com/gh) &nbsp;**Summary:** :trophy: This code is awesome&nbsp; | &nbsp;**Tags:** Elegant